instrumentation/all/opentelemetry-instrumentation-all.gemspec

```diff
@@ -61,6 +61,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'opentelemetry-instrumentation-sinatra', '~> 0.19.4'
   spec.add_dependency 'opentelemetry-instrumentation-trilogy', '~> 0.50.0'
 
+  spec.add_development_dependency 'active_model_serializers'
+  spec.add_development_dependency 'activesupport'
```

Comment:
These are showing up above in the run time dependencies and here in the dev dependencies.  Is there a reason for that?  Just want to make sure I'm not missing something nuanced here.

Reply:
I relocated them from the `Gemfile` to the `gemspec` so I assumed they were only needed for tests but now that you point it out I am going to double check if they are needed at all. 

I do not see them in the runtime dependencies. Where did you see them?

https://github.com/arielvalentin/opentelemetry-ruby/blob/remove-local-path-deps/instrumentation/all/opentelemetry-instrumentation-all.gemspec

Reply:
🤦‍♂️ , ignore me.  As it turns out `opentelemetry-instrumentation-active_support != activesupport`